examples/sample-app/application-template-dockerbuild.json

```diff
@@ -354,7 +354,7 @@
             "containers": [
               {
                 "name": "ruby-helloworld-database",
-                "image": "rhel8/mysql-80:latest",
+                "image": "image-registry.openshift-image-registry.svc/openshift/mysql:8.0-el8",
```

Comment:
pulling this through the internal registry means any test using this deploymentconfig will fail on clusters that don't have an internal registry(which is a concern since the internal registry is optional).  Is there a specific reason this is being updated to pull through the internal registry instead of from the upstream docker repository?  If this isn't explicitly part of a test for the internal registry/pullthrough, we probably should not do this like this.

Reply:
Oh, indeed. Moving back to WIP